Casks/r/rivet.rb

```diff
@@ -0,0 +1,28 @@
+cask "rivet" do
+  version "1.9.0"
+  sha256 :no_check
+
+  url "https://github.com/Ironclad/rivet/releases/download/app-v#{version}/Rivet.dmg"
+  name "rivet"
+  desc "Open-Source Visual AI Programming Environment"
+  homepage "https://github.com/Ironclad/rivet"
```

Comment:
```suggestion
  homepage "https://rivet.ironcladapp.com/"
```

After applying this change, the `url` will need a `verified` parameter added.